src/components/draft/Draft.tsx

```diff
@@ -204,7 +204,7 @@ class Draft extends React.Component<IProps, IState> {
 
                     {!this.state.simplifiedUI && <DraftIdInfo/>}
 
-                    {!this.state.simplifiedUI && <CivGrid civilisations={this.props.preset.civilisations}/>}
+                    <CivGrid civilisations={this.props.preset.civilisations}/>
```

Comment:
Why the change of mind?

Reply:
We will have to either remove the Simple UI toggle for Host and Guest Captains OR show the CivGrid in Simple UI. I chose the later thinking some Captains may want to stream the draft on their stream with Simple UI. May be I am over thinking this. 🙄

Reply:
There is also third option (which I did not think of earlier) – only show CivGrid on Simple UI for the Captains.

Reply:
> May be I am over thinking this. :roll_eyes: 

Feels like it :smile: 
Let's simply keep the CivGrid in the simple UI then, streamers can very easily crop that if they don't want to show it